lms/static/js/commerce/track_ecommerce_events.js

```diff
@@ -31,4 +31,6 @@
 
         return TrackECommerceEvents;
     });
-}).call(this, define || RequireJS.define);
+}).call(this,
+    typeof define === 'function' && define.amd ? define : RequireJS.define
+);
```

Comment:
@muselesscreator Does JS interpret this as ```(typeof define === 'function' && define.amd) ? define : RequireJS.define``` (note the parentheses)

Reply:
i think so